src/pyxplor/plot_time_series.py

```diff
@@ -0,0 +1,68 @@
+import pandas as pd
+import matplotlib.pyplot as plt
+import numpy as np
+
+def plot_time_series(input_df: pd.DataFrame, 
+
+                      date_column: str, 
+                      value_columns: list, 
+                      freq: str = 'D', 
+                      figsize: tuple = (10, 6),
+                      output_path: str = None, 
+                      super_title: str = "Time Series Analysis",
+                      super_title_font: int = 14) -> None:
+    """
+    Conducts exploratory data analysis on multiple time-series variables and generates line plot visualizations.
+
+    Parameters
+    ----------
+    input_df : pd.DataFrame
+        The DataFrame containing the time-series data. Expected to have at least two columns: 
+        one for dates and others for numeric values.
+        
+    date_column : str
+        The name of the column in `input_df` that contains date/time information. 
+        This column should be convertible to pandas datetime format.
+
+    value_columns : list
+        A list of names of the columns in `input_df` that contain the numeric values to be analyzed.
+
+    freq : str, optional
```

Comment:
If freq can only take on certain values, then you can specify as below
```suggestion
    freq : {"D", "W", "M"}, optional
```